stories/FormWithStepper/FormWithStepper.stories.tsx

```diff
@@ -107,4 +109,21 @@ const Template: StoryFn<Props> = args => {
   );
 };
 
+const Template: StoryFn<Props> = args => {
+  return <>{sampleFormWithStepper(() => {})}</>;
+};
+
+const Template1: StoryFn<Props> = args => {
```

Comment:
```suggestion
const TemplateWithScroll: StoryFn<Props> = args => {
```

or something similar

Reply:
Removed and created new story for hook